Frontend/ui-customer/src/Components/QuoteListModel/Controller.purs

```diff
@@ -60,4 +60,6 @@ type QuoteListModelState = {
   , selectedEstimatesObject :: CVC.Config
   , showAnim :: Boolean
   , animEndTime :: Int
+  , currentStage :: Stage
```

Comment:
not required, is it added on purpose ?

Reply:
was added on purpose only, not required anymore. I'll remove it.